src/style-spec/reference/v8.json

```diff
@@ -251,6 +256,11 @@
       "type": "string",
       "doc": "Contains an attribution to be displayed when the map is shown to a user."
     },
+    "volatile": {
+      "type": "boolean",
+      "default": false,
+      "doc": "Setting this flag to true indicates that the source tiles shall not be stored in the local cache." 
```

Comment:
@pozdnyakov we should update the documentation here too so that the language is the same everywhere

Reply:
Sure, I just applied your proposal (thanks for it!) to pull it locally :) with the latest patch this change is applied everywhere

Reply:
oh gotcha, thanks!